### PR TITLE
OFS-99: fixing bugs reported from Rafal

### PR DESCRIPTION
**Pull request by sniedzielski**
In that pull request I want to merge some changes connected with fixing bugs reported by Rafal Stencel @rstencel. 

Fixed issues:
- "orderBy" filter
- dateValidTo and dateValidFrom as an input fields
- locations uuid vs id (there is still primary key in Location as an ID)
- filtering old records vs active records (is_deleted=False or is_deleted=True) *

* I have to make another small pull request for small change in core module with filtering queryset. Unfortunately there is a problem with filtering is_deleted=False as a default value and in queries in frontend if somebody wants to retireve only deleted or only active records it should be done with using "is_deleted" filter during making query.

### CODE REVIEW

**Comment by sniedzielski** [comment]
https://github.com/openimis/openimis-be-core_py/pull/26 - another pull request with small change in core module related with filtering "is_deleted"

File: policyholder/gql/gql_mutations/input_types.py [hunk 1]
```diff
@@ -17,24 +17,25 @@ class PolicyHolderInputType(OpenIMISMutation.Input):
...
 
 class PolicyHolderInsureeInputType(OpenIMISMutation.Input):
     id = graphene.UUID(required=False)
-
     policy_holder_id = graphene.UUID(required=False)
-    insuree_id = graphene.UUID(required=False)
-    contribution_plan_bundle_id = graphene.UUID(requried=False)
-    last_policy_id = graphene.Int(required=False)
+    insuree_id = graphene.Int(required=False, name="insureeId")
+    contribution_plan_bundle_id = graphene.Int(requried=False, name="contributionPlanBundleId")
```
Comment (delcroip) [inline policyholder/gql/gql_mutations/input_types.py:29]: Contribution bundle will be created without int Id

**Review by delcroip — CHANGES_REQUESTED** [review]
See comments

File: policyholder/gql/gql_mutations/input_types.py (hunk 1 above)
Comment (sniedzielski) [reply policyholder/gql/gql_mutations/input_types.py:29]: Ok

**Comment by sniedzielski** [comment]
@delcroip - https://github.com/openimis/openimis-be-core_py/pull/27 - I also must fix this issue with django-dirty-fields. As a default django-dirty-fields isn't able to detect changes in models with foreign key if foreign key is changed. But with the special param function "is_dirty" starts to behave properly in such situations. I've detected this bug with Rafal during updating PolicyHolder with changing only locations Id . 

Please merge this one. It is really significant for working with models containing foreign keys.
Thx.

**Review by delcroip — APPROVED** [review]

**Pull request merged**